models/public/facenet-20180408-102900/README.md

```diff
@@ -73,6 +73,12 @@ An example of using the Model Converter:
 omz_converter --name <model_name>
 ```
 
+## Demo usage
+
+The model can be used in the following demos provided by the OpenVINO toolkit to show its capabilities:
+
+* [Face Recognition Python\* Demo](../../../demos/face_recognition_demo/python/README.md)
```

Comment:
probably smart classroom is also suitable

Reply:
Added